File: RecyclerView_PHP/app/src/main/java/com/example/recyclerview_php/Pelicula.java
package com.example.recyclerview_php;

public class Pelicula {

    public int ID;
    public String nombre;
    public String director;
    public String descripcion;

    public Pelicula(int ID, String nombre, String director, String descripcion) {
        this.ID = ID;
        this.nombre = nombre;
        this.director = director;
        this.descripcion = descripcion;

    }

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDirector() {
        return director;
    }

    public void setDirector(String director) {
        this.director = director;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }
}

File: Required for RecyclerView_PHP/peliculas/modelos/Pelicula.php
<?php

// Esta clase representa una pelicula

class Pelicula
{
    // Variables de clase
    public $ID, $nombre, $director, $descripcion;

    // Constructor
    public function __construct($nID, $nnombre, $ndirector, $ndescripcion)
    {
        $this->ID = $nID;
        $this->nombre = $nnombre;
        $this->director = $ndirector;
        $this->descripcion = $ndescripcion;
    }

    // Getter para ID
    public function getID()
    {
        return $this->ID;
    }

    // Getter para nombre
    public function getNombre()
    {
        return $this->nombre;
    }

    // Getter para director
    public function getDirector()
    {
        return $this->director;
    }

    // Getter para descripcion
    public function getDescripcion()
    {
        return $this->descripcion;
    }

    // Muestra los datos de la pelicula
    public function toString()
    {
        return
            [
                "ID" => utf8_encode($this->ID),
                "nombre" => utf8_encode($this->nombre),
                "director" => utf8_encode($this->director),
                "descripcion" => utf8_encode($this->descripcion),
            ];
    }
}
